packages/next/src/client/lib/console.ts

```diff
@@ -109,3 +109,33 @@ export function formatConsoleArgs(args: unknown[]): string {
 
   return result
 }
+
+// TODO: Infer type for `environmentName` if available.
+export function parseEnvironmentName(args: unknown[]): string | null {
```

Comment:
we already  duplicate this helper in `intercept-console.error.ts` called `matchReplayedError`, let's merge into one  that can return both error and label